### PR TITLE
Update README with a note on the Julia wrapper.

### PR DESCRIPTION
**Pull request by jmark**
**_Describe your changes here:_**

Title says it all. I added this note considering the [Helmholtz Incubator Software Award 2023](https://wiki.dlr.de/pages/viewpage.action?pageId=564241334) so that the jury sees that there is an ecosystem developing around t8code.

**_All these boxes must be checked by the reviewers before merging the pull request:_**

As a reviewer please read through all the code lines and make sure that the code is fully understood, bug free, well-documented and well-structured.

#### General
- [ ] The reviewer executed the new code features at least once and checked the results manually

- [ ] The code follows the [t8code coding guidelines](https://github.com/holke/t8code/wiki/Coding-Guideline)
- [ ] New source/header files are properly added to the Makefiles
- [ ] The code is well documented
- [ ] All function declarations, structs/classes and their members have a proper doxygen documentation
- [ ] All new algorithms and data structures are sufficiently optimal in terms of memory and runtime (If this should be merged, but there is still potential for optimization, create a new issue)

#### Tests
- [ ] The code is covered in an existing or new test case using Google Test

#### Github action

- [ ] The code compiles without warning in debugging and release mode, with and without MPI (this should be executed automatically in a github action)
- [ ] All tests pass (in various configurations, this should be executed automatically in a github action)

  If the Pull request introduces code that is not covered by the github action (for example coupling with a new library):
  - [ ] Should this use case be added to the github action?
  - [ ] If not, does the specific use case compile and all tests pass (check manually)

#### Scripts and Wiki

- [ ] If a new directory with source-files is added, it must be covered by the `script/find_all_source_files.scp` to check the indentation of these files.
- [ ] New Datatypes are added to t8indent_custom_datatypes.txt
- [ ] If this PR introduces a new feature, it must be covered in an example/tutorial and a Wiki article.

#### Licence

- [ ] The author added a BSD statement to `doc/` (or already has one)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 t8code (spoken as "tetcode") is a C/C++ library to manage parallel adaptive meshes with various element types.
 t8code uses a collection (a forest) of multiple connected adaptive space-trees in parallel and scales to at least one million MPI ranks and over 1 Trillion mesh elements.
-It is licensed under the GNU General Public License 2.0 or later. Copyright (c) 2015 the developers.
+It is licensed under the GNU General Public License 2.0 or later. Copyright (c) 2023 the developers.
```
Comment (holke) [inline README.md:11]: Please do not change the copyright message - especially if the PR is not related to Copyright.

You are right that we should update the year. Following https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html we should list all years where we did changes (so all from 2015 to 2023). But let's inform ourselves here on the best practices and then do a stand-alone PR.

Comment (jmark) [reply README.md:11]: Fair enough. I'll revert this change.

**Review by sandro-elsweijer — APPROVED** [review]
